content/scalability/docs/control-plane.md

```diff
@@ -93,3 +93,86 @@ autoscalingGroups:
+
+
+### Monitoring APF and Dropped Requests:
+
+To confirm if any requests are being dropped due to APF, the API Server metrics for ```apiserver_flowcontrol_rejected_requests_total``` can be monitored to check the impacted FlowSchemas and PriorityLevelConfigurations. For example, this metric shows that 100 requests from the service-accounts FlowSchema were dropped due to requests timing out in workload-low queues:
+
+```
+% kubectl get --raw /metrics | grep rapiserver_flowcontrol_rejected_requests_total
+apiserver_flowcontrol_rejected_requests_total{flow_schema="service-accounts",priority_level="workload-low",reason="time-out"} 100
+```
+To check how close a given PriorityLevelConfiguration is to receiving 429s or experiencing increased latency due to queuing, you can compare the difference between the concurrency limit and the concurrency in use. In this example, we have a buffer of 100 requests.
+
+```
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_request_concurrency_limit.*workload-low'
+apiserver_flowcontrol_request_concurrency_limit{priority_level="workload-low"} 245
+
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_request_concurrency_in_use.*workload-low'
+apiserver_flowcontrol_request_concurrency_in_use{flow_schema="service-accounts",priority_level="workload-low"} 145
+```
+To check if a given PriorityLevelConfiguration is experiencing queuing but not necessarily dropped requests, the metric for ```apiserver_flowcontrol_current_inqueue_requests``` can be referenced:
+```
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_current_inqueue_requests.*workload-low'
+apiserver_flowcontrol_current_inqueue_requests{flow_schema="service-accounts",priority_level="workload-low"} 10
+```
+
+Other useful Prometheus metrics include:
+- apiserver_flowcontrol_dispatched_requests_total
+- apiserver_flowcontrol_request_execution_seconds
+- apiserver_flowcontrol_request_wait_duration_seconds
+
+See the upstream documentation for a complete list of [APF metrics](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#observability).
+
```

Comment:
Maybe it's worth adding something like `If you are not sure which FlowSchema and PriorityLevelConfiguration your request matches, you can look the [response header added by APF](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#response-headers)`.

Reply:
I wasn't aware of that feature but we set the verbosity as 2 for when a customer enables control plane logging so they wouldn't have access to these events.